apps/src/acemode/annotationList.js

```diff
@@ -52,6 +52,15 @@ module.exports = {
   },
 
   /**
+   * Return the array of lintAnnotations
+   *
+   * @returns {!Array} annotations from most recent linting pass
+   */
+  getJSLintAnnotations: function () {
```

Comment:
Good point. We're discussing allowing the level builder to filter the warnings based on type (instead of blocking based on the presence of errors/warnings), so I left this function more general for now.